src/widgets/FormControl.js

```diff
@@ -169,6 +167,36 @@ const FormControlComponent = ({
       }
     );
 
+  const Buttons = useCallback(
```

Comment:
Yeah, as commented, this "solution" is more of a temporary cludge to get things working to make sure the whole thing is feasible. For that reason I think best to leave it for now (if OK with you). 

I'll probably need to think of a better way of doing it. I agree with the props can be more granular, but my thinking was that it might be a bit of a smell that we have to specify props to a component to tell it what children to render (I think also, even with flex, there are often niggly issues with margins etc. which will prob require passing style props). 

I was thinking perhaps of separating out the buttons from component altogether? In any case, I think maybe something to return to in a new issue once the rest of the UI/functionality is in decent shape?